src/sonic-yang-models/yang-templates/sonic-policer.yang.j2

```diff
@@ -0,0 +1,126 @@
+{% if yang_model_type == "cvl" %}
+/* this is sonic cvl yang model */
+{% else %}
+/* this is sonic py yang model */
+{% endif %}
+module sonic-policer {
+
+	yang-version 1.1;
+
+	namespace "http://github.com/Azure/sonic-policer";
+	prefix policer;
+
+	import sonic-types {
+		prefix stypes;
+	}
+
+	description "Policer YANG Module for SONiC OS";
+
+	revision 2022-02-03 {
+		description
+			"First Revision";
+	}
+
+	container sonic-policer {
+		container POLICER {
+			list POLICER_LIST {
+				key name;
+
+				leaf name {
+					type string;
+					description "Policer name";
+				}
+
+				leaf meter_type {
+					mandatory true;
+					type stypes:meter_type;
+					description "Policer meter type";
+				}
+
+				leaf mode {
+					mandatory true;
+                                        type stypes:policer_mode;
+					description "Policer mode";
+				}
+ 
+  				leaf color {
+                                        type stypes:policer_color_source;
+					description "Policer color Source";
+				} 
+
+				leaf cir {
+					type uint64;
+					default 0;
+					description
+						"Committed information rate for the dual-rate token
+						bucket policer.  This value represents the rate at which
+						tokens are added to the primary bucket.";
+				}
+
+				leaf cbs {
+					must "((current()/../cir) and (current()/../cir > 0))" {
+						error-message "cbs can't be configured without cir.";
+					}
+					must "(current() >= current()/../cir)" {
```

Comment:
Please refer to https://datatracker.ietf.org/doc/html/rfc4115.
The CBS and EBS are measured in bytes and must configure to be greater than the expected maximum length of the incoming PDU.  The CIR and EIR are both measured in bits/s.
I think we need to convert unit for comparing.

Reply:
Please refer to my earlier comments regarding the units.

Reply:
Can you update the descriptions for CIR, CBS, PIR and PBS to add unit?